tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -1042,6 +1056,33 @@ public function theUserDeclinesTheOfferedFederatedShares() {
 		}
 	}
 
+	/**
+	 * @Given the administrator has :action public link quick action
+	 *
+	 * @param string $action
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function enablePublicLinkQuickAction(string $action):void {
+		if ($action === "enabled") {
+			SetupHelper::setSystemConfig(
+				'sharing.showPublicLinkQuickAction',
+				'true',
```

Comment:
```suggestion
				($action === "enabled") ? 'true' : 'false',
```
you can use the ternary operator just for this arg since all other args are the same.